File: Examples/VisualBasic/Quick-Start/Doc2Pdf.vb
Imports Microsoft.VisualBasic
Imports System.IO

Imports Aspose.Words

Public Class Doc2Pdf
    Public Shared Sub Run()
        ' The path to the documents directory.
        Dim dataDir As String = RunExamples.GetDataDir_QuickStart()

        ' Load the document from disk.
        Dim doc As New Document(dataDir & "Template.doc")

        ' Save the document in PDF format.
        doc.Save(dataDir & "Doc2PdfSave Out.pdf")

        Console.WriteLine(vbNewLine + "Document converted to PDF successfully." + vbNewLine + "File saved at " + dataDir + "Doc2PdfSave Out.pdf")
    End Sub
End Class

File: Plugins/Aspose_Words_NET_for_PHP/src/aspose/words/quickstart/Doc2Pdf.php
<?php
/**
 * Created by PhpStorm.
 * User: assadmahmood
 * Date: 30/06/15
 * Time: 10:45 AM
 */

namespace Aspose\Words\QuickStart;


class Doc2Pdf {

    public static function run($dataDir=null)
    {
        if(is_null($dataDir)) die("Data Directory Undefined");

        $comHelper = new \COM("Aspose.Words.ComHelper");

        $doc = $comHelper->Open($dataDir."/Template.doc");

        // Save the document in PDF format.
        $doc->Save($dataDir . "/Doc2PdfSave Out.pdf");

        print "Document converted to PDF successfully.\nFile saved at " . $dataDir . "Doc2PdfSave Out.pdf" . PHP_EOL;

    }

} 